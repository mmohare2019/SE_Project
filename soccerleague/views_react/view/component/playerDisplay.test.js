import "react-native";
import "react";
import PlayerDisplay from "./playerDisplay";
import { test, expect } from "@jest/globals";
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test("Testing team display field", function () {
    const component = renderer
    .create(<PlayerDisplay onPress={()=>1}/>).toJSON();
    expect(component).toMatchSnapshot();
}); 