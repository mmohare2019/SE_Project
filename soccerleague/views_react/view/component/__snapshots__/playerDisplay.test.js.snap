// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing team display field 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#d4d4d4",
      "marginHorizontal": 16,
      "marginVertical": 8,
      "opacity": 1,
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
     First name: 
     
  </Text>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
     Last name: 
     
  </Text>
  <Text
    style={
      Object {
        "fontSize": 16,
      }
    }
  >
     Email: 
     
  </Text>
</View>
`;
